coalib/output/ConsoleInteraction.py

```diff
@@ -164,35 +158,47 @@ def print_lines(console_printer,
     :param sourcerange:     The SourceRange object referring to the related
                             lines to print.
     """
+
     for i in range(sourcerange.start.line, sourcerange.end.line + 1):
         console_printer.print(format_lines(lines='', line_nr=i),
                               color=FILE_LINES_COLOR,
                               end='')
+
         line = file_dict[sourcerange.file][i - 1].rstrip("\n")
-        tab_width = int(section.get('tab_width', 4))
-        s = SpacingHelper(tab_width)
-        tab_dict = dict(s.yield_tab_lengths(line))
+        # Get suitable lexer depending on filename's extension and if not found
+        # fallback to TextLexer().
+        try:
+            lexer = get_lexer_for_filename(sourcerange.file)
+        except ClassNotFound:
+            lexer = TextLexer()
+        s = SpacingHelper.DEFAULT_TAB_WIDTH
+        lexer.add_filter(VisibleWhitespaceFilter(spaces="•", tabs=True,
+                                                 tabsize=s))
```

Comment:
I did it because unable follow the 80 chars rule otherwise. Also it was done [here](https://github.com/coala-analyzer/coala/blob/master/coalib/output/ConsoleInteraction.py#L173) so I thought I can use this style?